<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tunnel_inspectionClass</class>
 <widget class="QMainWindow" name="tunnel_inspectionClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1272</width>
    <height>908</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>隧道巡检</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="picture_pushbutton">
          <property name="text">
           <string>图像加载</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="load_pushbutton">
          <property name="text">
           <string>加载工控机1文件夹</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="load_lineedit">
          <property name="text">
           <string>J:/1231bak1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="load_pushbutton_2">
          <property name="text">
           <string>加载工控机2文件夹</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="load_lineedit_2">
          <property name="text">
           <string>J:/1118bak2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="path_pushbutton">
          <property name="text">
           <string>保存原图路径</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="path_lineedit">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="result_path_pushbutton">
          <property name="text">
           <string>保存结果路径</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="result_path_lineedit">
          <property name="text">
           <string>H:/tunnel_test/</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="save_pushbutton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>开始保存</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="save_radio_button">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>保存原图</string>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radio_button">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>显示回放图像</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="start_pushbutton">
          <property name="text">
           <string>开始回放检测</string>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="time_label">
          <property name="text">
           <string>执行时间</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="progress_bar">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="oringinal_label">
        <property name="minimumSize">
         <size>
          <width>520</width>
          <height>800</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border:2px solid darkblue;
border-radius:0px;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1272</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../icon/tunnel_inspection.qrc"/>
 </resources>
 <connections/>
</ui>
